openverse_catalog/dags/common/slack.py

```diff
@@ -220,24 +220,32 @@ def send_message(
     s.send(text)
 
 
-def on_failure_callback(context: dict) -> None:
+def should_send_message(http_conn_id=SLACK_NOTIFICATIONS_CONN_ID):
     """
-    Send an alert out regarding a failure to Slack.
-    Errors are only sent out in production and if a Slack connection is defined.
+    Returns true if a Slack connection is defined and we are in production (or
+    the force alert is set).
     """
     # Exit early if no slack connection exists
-    hook = HttpHook(http_conn_id=SLACK_ALERTS_CONN_ID)
+    hook = HttpHook(http_conn_id=http_conn_id)
     try:
         hook.get_conn()
     except AirflowNotFoundException:
-        return
+        return False
 
     # Exit early if we aren't on production or if force alert is not set
     environment = Variable.get("environment", default_var="dev")
     force_alert = Variable.get(
         "force_slack_alert", default_var=False, deserialize_json=True
     )
-    if not (environment == "prod" or force_alert):
+    return environment == "prod" or force_alert
```

Comment:
I pulled this logic out from `on_failure_callback` since we'll want to make sure that the TSV load complete message doesn't fire outside of production. I want to call out that this might be an overuse of the `force_slack_alert` environment variable, using it to force slack _notifications_ as well. Would it be worthwhile to have a separate flag, or I suppose to rename the existing one (`force_slack_message`?)

Reply:
That's a great call! +1 for having it be a separate function. And agreed, I think `force_slack_message` or maybe even `slack_message_override`? The name is from before #359 when we only had slack alerts.

Reply:
Oh I like `slack_message_override` even better! Updated, thanks for the context 👍